services/user.py

```diff
@@ -0,0 +1,52 @@
+from django.shortcuts import get_object_or_404
+
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> User:
+
+    user_data = {
+        "username": username,
+        "password": password
+    }
+    if email:
+        user_data["email"] = email
+    if first_name:
+        user_data["first_name"] = first_name
+    if last_name:
+        user_data["last_name"] = last_name
+
+    some_user = User.objects.create_user(**user_data)
```

Comment:
Ensure that the `create_user` method hashes passwords correctly. Storing passwords in plain text is a security risk. Use `set_password` method to hash the password before saving the user.